lms/djangoapps/lti_provider/tests/test_views.py

```diff
@@ -112,10 +114,10 @@ def test_session_contents_after_launch(self, _run):
         properly stored in the session
         """
         request = build_launch_request()
-        views.lti_launch(request, COURSE_PARAMS['course_id'], COURSE_PARAMS['usage_id'])
+        views.lti_launch(request, str(COURSE_PARAMS['course_key']), str(COURSE_PARAMS['usage_key']))
```

Comment:
Can we use `unicode` instead of `str` here? All these parameters will be unicode objects when they come in via Django during actual requests, right?